modules/fileout_netcdf/FONcArray.cc

```diff
@@ -40,6 +41,7 @@
 #include "FONcMap.h"
 #include "FONcUtils.h"
 #include "FONcAttributes.h"
+#include <algorithm>
```

Comment:
Not a huge deal, but I like to keep the different types of headers grouped together. C++ headers, then C headers, then libdap, bes and finally handler.